libkbfs/bserver_measured.go

```diff
@@ -66,6 +66,16 @@ func (b BlockServerMeasured) Put(ctx context.Context, tlfID tlf.ID, id kbfsblock
 	return err
 }
 
+// PutAgain implements the BlockServer interface for BlockServerMeasured.
+func (b BlockServerMeasured) PutAgain(ctx context.Context, tlfID tlf.ID, id kbfsblock.ID,
+	context kbfsblock.Context, buf []byte,
+	serverHalf kbfscrypto.BlockCryptKeyServerHalf) (err error) {
+	b.putTimer.Time(func() {
```

Comment:
We usually have one timer per RPC, so I slightly prefer making a `putAgainTimer` for this.  But if you want to combine the two for a specific reason, I don't feel too strongly against it.